cmd/agent/app/run.go

```diff
@@ -403,9 +406,6 @@ func StartAgent() error {
 	// Append version and timestamp to version history log file if this Agent is different than the last run version
 	util.LogVersionHistory()
 
-	// create and setup the Autoconfig instance
-	common.LoadComponents(common.MainCtx, config.Datadog.GetString("confd_path"))
-
```

Comment:
This is a terrifying distance to move this stanza, but .. I guess, if it works, it works?

Reply:
If it works, it works. To the best of my knowledge, `LoadComponents` doesn't seem to depend on anything set up between the old and new locations.